esp-alloc/Cargo.toml

```diff
@@ -1,13 +1,14 @@
 [package]
-name         = "esp-alloc"
-version      = "0.7.0"
-edition      = "2021"
-rust-version = "1.84.0"
-description  = "A heap allocator for Espressif devices"
-keywords     = ["allocator", "embedded", "esp32", "espressif", "memory"]
-categories   = ["embedded", "memory-management", "no-std"]
-repository   = "https://github.com/esp-rs/esp-hal"
-license      = "MIT OR Apache-2.0"
+name          = "esp-alloc"
+version       = "0.7.0"
+edition       = "2021"
+rust-version  = "1.84.0"
+description   = "A heap allocator for Espressif devices"
+documentation = "https://docs.espressif.com/projects/rust/"
```

Comment:
Tip to reviewers: "cog icon -> Hide whitespace" (or just `&w=1` in the URL) hides a lot of noise.

Reply:
We should try and get `https://docs.espressif.com/projects/rust/esp-alloc/` to redirect to the latest version, it would be so much better than just showing the landing page.

Reply:
Could you open an issue about that? I can look into it.